install/helm/agones/templates/crds/_gameserverstatus.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
As this is a new file this should be 2020

Reply:
:thinking: this was technically a file rename, not a file create - should the licence change?

Reply:
Oh wait, I'm totally wrong. Ignore me. I'll make this change.